### PR TITLE
Fix broken link to RHEL 8 doc

### PR DESCRIPTION
**Pull request by apinnick**
Also updated Vale workflow because it was not running.

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [x] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: .github/workflows/vale.yml [hunk 1]
```diff
@@ -3,8 +3,7 @@ name: Vale
 on:
   pull_request:
     paths:
-      - 'guides/common/*.adoc'
-      - '.github/workflows/vale.yml'
+      - '**.adoc'
```
Comment (ekohl) [inline .github/workflows/vale.yml:6]: Please separate these changes, at least to its own commit.

Comment (apinnick) [reply .github/workflows/vale.yml:6]: Done.

File: .github/workflows/vale.yml [hunk 2]
```diff
@@ -3,8 +3,7 @@ name: Vale
 on:
   pull_request:
     paths:
-      - 'guides/common/*.adoc'
-      - '.github/workflows/vale.yml'
```
Comment (maximiliankolb) [inline .github/workflows/vale.yml:7]: If not absolutely required, I'd like to keep this in to ensure vale runs if we change the vale GHA itself.

**Review by maximiliankolb — COMMENTED** [review]
one suggestion; URL for RHEL8 works :heavy_check_mark:

File: .github/workflows/vale.yml (hunk 2 above)
Comment (apinnick) [reply .github/workflows/vale.yml:7]: Not sure I understand. You want me to revert these lines? I thought that they were causing the check not to run.

Comment (ekohl) [inline .github/workflows/vale.yml:7]: Like @maximiliankolb I think it's good to run a workflow if you make changes to it. I think it should be:
```suggestion
      - 'guides/common/**.adoc'
      - '.github/workflows/vale.yml'
```

Comment (maximiliankolb) [reply .github/workflows/vale.yml:7]: Only line 7; I'd always run an action if the action itself is changed.

Comment (apinnick) [reply .github/workflows/vale.yml:7]: Will give this a try.

**Review by maximiliankolb — APPROVED** [review]
Now sure why the change in `vale.yml` is necessary. I cannot find the result for your PR, only the result for all of foreman-documentation. vale seems to run though :heavy_check_mark:

**Comment by apinnick** [comment]
Merged and cherrypicked:

b358de4480..f707d9cbdd  3.11 -> 3.11
02d23303af..7e66719dc8  3.10 -> 3.10

**Pull request merged**